netbox/dcim/choices.py

```diff
@@ -911,6 +922,20 @@ class InterfaceTypeChoices(ChoiceSet):
                 (TYPE_400GE_OSFP, 'OSFP (400GE)'),
             )
         ),
+        (
+            'Ethernet (backplane)',
```

Comment:
Per your comment [here](https://github.com/netbox-community/netbox/issues/10356#issuecomment-1245588171) we should call these out as 802.3ap types.
```suggestion
            'Backplane (802.3ap)',
```